src/PlusDataCollection/CMakeLists.txt

```diff
@@ -1498,16 +1498,21 @@ IF(PLUS_USE_WINPROBE_VIDEO)
 
   LIST(APPEND ${PROJECT_NAME}_LIBS
     ${WINPROBESDK_DIR}/${Bitness}/$<$<CONFIG:Debug>:Debug>$<$<NOT:$<CONFIG:Debug>>:Release>/UltraVisionManagedDll${CMAKE_STATIC_LIBRARY_SUFFIX}
```

Comment:
Does $<CONFIG:Debug> and similar work for multi config generators like xcode or msvc?

Reply:
These wrappers are compiled primarily (or even only) using MSVC. I expect that this would work for XCode too.